ensime-helm.el

```diff
@@ -0,0 +1,36 @@
+;;; ensime-helm.el
+;;
+;;;; License
+;;
+;;     Copyright (C) 2015 Aemon Cannon
```

Comment:
pretty sure Aemon didn't write this :smile:

Please follow the header pattern from `ensime.el` and